### ISSUE TITLE
Improvements to submission pdf (1.3.1 info and relationships)

### ISSUE THREAD

**Issue by LaurensBurger** [open-formulieren/open-forms#6367]
### Thema / Theme

Frontend

### Omschrijving / Description

The submission pdf is missing the structure that is present in the "overview". 
While fields and values are "linked":

<img width="251" height="200" alt="Image" src="https://github.com/user-attachments/assets/b24a5aec-a6f2-4e41-8921-60547708cc69" />

The pdf does not include any of these a11y helpers. 

Found issues:

> Container-tags (NonStruct, Sect, Div)
> 
> Dit zijn tags waar andere tags in mogen staan. Er mag nooit tekst rechtstreeks in de container-tags staan. Dat is hier wel het geval. Bovendien blijkt uit de Screen Reader Preview dat er tekst dubbel in de codelaag staat.
> 
> Dit kan leiden tot ernstige problemen bij het voorlezen en navigeren door een screenreader. Dit hangt mede af van de combinatie van de gebruikte screenreader en PDF-reader. Dus een controle met een bepaalde combinatie, zoals die van NVDA/Acrobat zegt nog niets. Een goed opgebouwde codelaag zal voor alle combinaties goed werken.
> 
> Alinea’s (P)
> 
> Alle gewone tekst hoort in P-tags te staan waarbij een zichtbare alinea in één P-tag hoort te staan. In deze PDF is dat maar zo hier en daar het geval. De meeste tekst staat buiten de P-tags. De P-tag wordt als zodanig herkend door een screenreader. De gebruiker kan daardoor snel door de alinea’s navigeren. Dat kan nu niet.
> 
> Koppen (H1, H2, H3, H4, H5, H6)
> 
> In deze PDF staan alleen de koppen op de eerste twee niveaus, H1 en H2 goed in de codelaag. In de zichtbare tekst bevinden zich veel meer koppen, maar die staan niet in een tag of ze staan in de verkeerde tag, zoals in een P-tag, samen met andere die wel in die P-tag hoort (“VerklaringDit verklaar ik:”).
> 
> Lijsten
> 
> De lijsten zijn gedeeltelijk goed opgebouwd in de codelaag, maar het opsommingsteken staat niet in de Lbl-tag en deze staat in de codelaag achter de tekst. Dat wordt dus in de verkeerde volgorde voorgelezen.
> 
> Links
> 
> De links zijn niet bereikbaar met het toetsenbord omdat ze niet goed opgebouwd zijn in de PDF-structuur, los van de codelaag. In de codelaag staan ze ook niet goed omdat de kern van de link, de Link-OBJR-tag ontbreekt.
> 
> Paginanummers
> 
> De paginanummers staan in de codelaag, maar dat is niet de bedoeling. Ze horen als decoratief (artefact) te worden gemarkeerd. De zichtbare nummers moeten overeenkomen met de paginanummers in de PDF-reader zelf. Dit is het geval in deze PDF.
>
